### ISSUE TITLE
Standardize on a configuration format (actions use ini, snips daemons uses toml)

### ISSUE THREAD

**Issue by hcooper** [snipsco/snips-actions-templates#10]
The `/etc/snips.toml` file is where a lot of central configuration options live for snips (e.g. which MQTT host to use). The file is in `toml`[1] format.

The snips actions-template system presumes that the action's config file will be a `ini` file (and is read by the `ConfigParser` module).

`ConfigParser` doesn't like importing `toml` files. For instance is throws `ParsingError()` when there's a space at the start of the line, and doesn't expect strings to be within quotes.

Using two different configuration systems means configs can't be shared (with a lot of manually-added boilerplate to read the `toml` file). It would be good to standardize, so that actions can easily read the same configs the snips daemons are using.

[1] https://en.wikipedia.org/wiki/TOML

**Comment by maxbachmann** [comment]
Yes it would be a lot better when the template takes the mqtt values from the config. Especially now that hermes does support the other mqtt options like tls aswell.
That way the user could change the settings for his setup without having to change things inside the actions.

**Comment by maxbachmann** [comment]
@anthonyray shouldn´t this be fixed with #18 ?
